app/models/metric/capture.rb

```diff
@@ -78,7 +72,10 @@ def self.perf_capture_timer(zone = nil)
 
       h
     end
+    targets_by_rollup_parent
+  end
 
+  def self.calc_tasks_by_rollup_parent(targets_by_rollup_parent)
```

Comment:
Can these new methods be made into private methods?  They shouldn't be part of the public interface, I don't think.